public/give.html

```diff
@@ -9,6 +9,8 @@
   gtag('js', new Date());
   gtag('config', 'UA-161442455-1');
   </script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/airbrake-js/1.6.7/client.min.js"></script>
```

Comment:
should we also add to stats / whoweare? Stats has some js, I don't believe whoweare does

Reply:
Maybe there should be a separate channel for this.

Reply:
We need to put all the critical js in a common js that is included in all the pages. We shouldn't have to remember to include it on each page (esp as I anticipate the new UX to give us more pages).